docs/swagger/spec.yml

```diff
@@ -20,8 +20,12 @@ consumes:
 tags:
```

Comment:
I don't think so -- this is meant for everything that has a `/api/` path in the URL.